x-pack/plugins/session_view/public/hooks/use_eui_theme.ts

```diff
@@ -29,6 +30,7 @@ export const useEuiTheme = (...props: EuiThemeProps): EuiThemeReturn => {
     const extraEuiVars: ExtraEuiVars = {
       // eslint-disable-next-line @typescript-eslint/naming-convention
       euiColorVis6_asText: shade(themeVars.euiColorVis6, 0.335),
+      buttonsBackgroundNormalDefaultPrimary: '#006DE4',
```

Comment:
Is this a color not provided by the eui theme? 👀

Reply:
yeah that seemed strange to me as well, but as you can see [here](https://www.figma.com/file/3v90FFYIKKZ7aIPxj2fxiH/%5B8.2%5D-%5BSession-viewer%5D-Session-View-for-Sessions-and-Processes?node-id=3198%3A382800), it uses a color that is meant to be the Background color of the primary button so that should be a valid color in the Design library according to the [buttons spec](https://www.figma.com/file/RzfYLj2xmH9K7gQtbSKygn/Elastic-UI?node-id=16773%3A71), but when scanning for the HEX or the RGB I couldn't find any matching, even tried combinations of `shade` or `lightness`, so I think that might be due to the fact that not all colors on the Design library were translated to JS, so we can use our euiTheme override to hold these values meanwhile.